e2e/Makefile

```
@@ -23,16 +24,43 @@ bench-governance-matrix: ## Benchmark governance query latency (full 3x3 custome
 
 .PHONY: test-local
 test-local: ## Run tests against local openmeter
+	$(call print-target)
+	$(MAKE) test-local-base
+	$(MAKE) test-local-credits-disabled
+
+.PHONY: test-local-credits-disabled
+test-local-credits-disabled: ## Run credits-disabled e2e tests against local openmeter
```

checkmake (0.3.2)

[warning] Target body for "test-local-credits-disabled" exceeds allowed length of 5 lines (6).

(maxbodylength)